### ISSUE TITLE
Protoparse cannot parse group inside oneof

### ISSUE THREAD

**Issue by pedgeio** [jhump/protoreflect#256]
This is actually valid Protobuf:

```proto
// a.proto

syntax = "proto2";

message Foo {
  oneof bar {
    group Baz = 1 {
      optional int64 bat = 2;
    }
  }
}
```

- `protoc a.proto -o /dev/null` completes successfully
- `goprotoc a.proto -o /dev/null` returns `a.proto:7:19: syntax error: unexpected '{', expecting ';' or '['`

**Comment by jhump** [comment]
Weird. I did not know that they extended group support to oneofs. I guess I should read that part of the protoc code more closely...

I think it should be pretty easy to fix, luckily.

**Comment by jhump** [comment]
Fixed in #262
